src/CodingMilitia.PlayBall.Shared.StartupTasks/DbInitializer.cs

```diff
@@ -9,17 +9,18 @@ public class DbInitializer<T> : IAsyncInitializer where T : DbContext
     {
         private IHostingEnvironment _hostingEnvironment;
         private DbContext _dbContext;
+        private string _environment;
 
-        public DbInitializer(T dbContext, IHostingEnvironment host)
+        public DbInitializer(T dbContext, IHostingEnvironment host, string environment)
```

Comment:
Instead of getting a `string`, we should get an instance of something like `DbInitializerSettings`.
Then it's easy to register in the DI container as a strongly typed configuration (see `ConfigurePOCO`).

Reply:
noted 😊